datashuttle/utils_mod/canonical_configs.py

```diff
@@ -97,18 +105,19 @@ def check_dict_values_and_inform_user(config_dict):
         )
 
     if config_dict["connection_method"] not in ["ssh", "local_filesystem"]:
-        utils.raise_error("connection method must be ssh or local")
+        utils.raise_error("connection method must be ssh or local_filesystem")
```

Comment:
**Incredibly** pedantic, but I'd make sure messages were capitalised, and had punctuation etc.